Composer/packages/lib/code-editor/demo/webpack.config.demo.js

```diff
@@ -103,6 +105,16 @@ module.exports = {
     path: path.resolve('./demo/dist'),
   },
 
+  node: {
```

Comment:
We shouldn't just empty out the package.

We should find the root cause and not importing native dependencies in client code.